### ISSUE TITLE
ListenBrainz support

### ISSUE THREAD

**Issue by jliuhtonen** [jliuhtonen/blu-hawaii#37]
It would be nice to support ListenBrainz in addition to Last.fm as listening data "sink".

**Comment by SatanicMechanic** [comment]
Perhaps libre.fm as well

**Comment by jliuhtonen** [comment]
Why not! Could you @SatanicMechanic create another issue about that so we could address that request separately?

**Comment by SatanicMechanic** [comment]
I went to look up info about the libre API and discovered that libre is in limbo and doesn't appear to have docs online.  So might be better to wait on that.

But for listenbrainz, I saw that they have a legacy [lastfm-compatible API ](https://listenbrainz.readthedocs.io/en/latest/users/api-compat.html#last-fm-api)so might be a shortcut?

**Comment by zquestz** [comment]
Don't think we need this anymore.

https://www.reddit.com/r/lastfm/comments/1gjyguz/listenbrainz_new_lastfm_sync_option/

**Comment by jliuhtonen** [comment]
> Don't think we need this anymore.
> 
> https://www.reddit.com/r/lastfm/comments/1gjyguz/listenbrainz_new_lastfm_sync_option/

While it is nice to be able to connect Last.fm to ListenBrainz, for me the point would be to be able to choose or change the service used to submit listening data to.
